components/collection/drop/GenerativeLayout.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div class="unlockable-container">
+    <div class="container is-fluid border-top pt-6">
+      <div class="columns is-desktop">
+        <div class="column is-half-desktop mobile-padding">
+          <UnlockableCollectionInfo
+            :collection-id="collectionId"
+            :description="description" />
+          <hr />
```

Comment:
maybe margin/padding instead?

Reply:
it was already there, but we wouldn't have the divider then
![CleanShot 2023-12-27 at 14 31 56@2x](https://github.com/kodadot/nft-gallery/assets/44554284/08021bf5-2ea0-4955-81e1-f0b3d215d232)

Reply:
ok